src/main/kotlin/no/nav/familie/baks/soknad/api/controllers/SøknadController.kt

```diff
@@ -24,7 +25,7 @@ import org.springframework.web.bind.annotation.RestController
 class SøknadController(private val mottakClient: MottakClient) {
 
     @PostMapping("/soknad/v8")
-    fun søknadsmottakV8(@RequestBody(required = true) søknad: SøknadV8): ResponseEntity<Ressurs<Kvittering>> {
+    fun søknadsmottakBarnetrygdV8(@RequestBody(required = true) søknad: BarnetrygdSøknadV8): ResponseEntity<Ressurs<Kvittering>> {
```

Comment:
Bare pirk, men kanskje vi bare kan kalle metoden: `søknadsmottakBarnetrygd` (uten V8) ? For å fremheve at det er denne som er gjeldende 🤔